lib/screen/main/payment_method_item.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:showcase_flutter_build_web_desktop/extension/context_extension.dart';
+import 'package:showcase_flutter_build_web_desktop/theme/dimen.dart';
+
+import '../../gen/colors.gen.dart';
+
+class PaymentMethodItem extends StatefulWidget {
```

Comment:
I think we can keep the `StatelessWidget` because we don't have any case to handle state here as we pass the values and control them in the parent widget(`main_screen`) 🙏 
```suggestion
class PaymentMethodItem extends StatelessWidget {
```